### ISSUE TITLE
saving as braille seems broken on macos

### ISSUE THREAD

**Issue by sthibaul** [mlang/freedots#7]
As discussed on https://www.freelists.org/post/liblouis-liblouisxml/Automatic-translation-of-MusicXML-to-braille-music,16 it seems saving as braille is broken on MacOS: the save-as dialog box only shows the xml choice, and not the others.

**Comment by judithhinlung** [comment]
Yes unfortunately the Swing Filechooser is not very compatible on OS X, so the file type options are not displaying correctly.  I am working on a pull request to fix this issue as well as make Freedots friendlier to use on Mac.
